Classes/Service/Extraction/AbstractExtractionService.php

```diff
@@ -323,7 +325,12 @@ protected function remapServiceOutput(array $data, array $mapping = null)
 
             $value = null;
             foreach ($alternativeKeys as $dataKey) {
-                list($compoundKey, $processor) = explode('->', $dataKey);
+                if (str_contains($dataKey, '->')) {
```

Comment:
This doesn't make it compatible as `str_contains` requires PHP 8, thus it breaks with PHP 7.x

Reply:
solved